operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -466,6 +467,7 @@ spec:
 #      in_cluster_url: ""
 #      namespace_selector: true
 #      url: ""
+#      whitelist_istio_system: ["jaeger-query", "istio-ingressgateway"]
```

Comment:
you need to add this to the operator's defaults yaml in the kiali-deploy role: add it after this line: https://github.com/kiali/kiali/blob/master/operator/roles/kiali-deploy/defaults/main.yml#L130

Reply:
omg I always forget this :( sorry